msteams-platform/bots/streaming-ux.md

```diff
@@ -0,0 +1,586 @@
+
+2. **Set informative message**:
+
+    You can define just one informative message for your AI-powered bot. This message appears for the user every time the bot sends an update. Here are some examples for informative messages that you can set in your app:
+
+    - **Scanning through documents**
+    - **Summarizing content**
+    - **Finding relevant work items**
+
+    The following example shows the information updates in an AI-powered bot:
+
+    :::image type="content" source="../assets/images/bots/streaming-ai-info-update.png" alt-text="Image shows information updates streaming in an AI bot.":::
+
+3. **Format the final streamed message**:
+
+    Using AI SDK, text messages and simple markdown can be formatted while they're being streamed. However, for Adaptive Cards, images, or rich HTML, the formatting can be applied once the final message is complete. The bot can send attachments only in the final streamed chunk.
+
+    The following example shows the streaming response in an AI-powered bot:
+
+    :::image type="content" source="../assets/images/bots/ai-response-streaming.png" alt-text="Image shows streaming responses in an AI bot.":::
+
+    The following example shows the final streamed response in an AI-powered bot:
+
+    :::image type="content" source="../assets/images/bots/ai-final-stream-message.png" alt-text="Image shows the final streamed response in an AI bot.":::
+
+4. **Enable AI-powered features for final message**:
+
+    You can enable the following AI-powered features for the final message sent by the bot:
+<br>
+
+    - **Citations**: The Teams AI library automatically includes citations in the bot's responses. It provides references for the sources that the bot used to generate the response. It allows users to refer to the source through in-text citations and references.
+    - **Sensitivity Label**: Use sensitivity label to help users understand the confidentiality of a message.
+    - **Feedback loop**: This allows users to provide positive or negative feedback on the bot messages.
+    - **Generated by AI**: The Teams AI library automatically includes a **Generated by AI** label in the bot's responses. This label helps users identify that a message was generated using AI.
+
+    For more information about formatting AI bot messages, see [bot messages with AI-generated content](how-to/bot-messages-ai-generated-content.md).
+
+### Configure streaming bot messages
+
+Follow these steps to configure streaming bot messages:
+
+1. **Enable streaming for AI bot**:
+
+    a. Use the `DefaultAugmentation` class in the `config.json` file, and in one of the following main application classes of your bot app:
+      - For a C# bot app: Update `program.cs`.
+      - For a JavaScript app: Update `index.ts`.
+      - For a Python app: Update `bot.py`.
+
+    b. Set `stream` to true in the `OpenAIModel` declaration.
+1. Configure the following AI-powered features:
```

Comment:
These are not powered by AI features. Please update to 

```
2. Additional Configurations
**Set informative message**: Specify the informative message in the ActionPlanner declaration using the StartStreamingMessage configuration.

**Add attachments to the final streamed message**: Set attachments in the final chunk using the EndStreamHandler within the ActionPlanner declaration.

**Enable the feedback loop**: Set the feedback loop toggle in the AIOptions object within the app declaration. For a bot app built using Python, set the feedback loop toggle in the ActionPlannerOptions object.
```

Reply:
Updated as discussed.